pages/_app.tsx

```diff
@@ -23,22 +25,25 @@ export default function App({ Component, pageProps }: AppProps) {
       await getStaticAsset('logos').then((l: ISupporterFunderLogos) => setLogos(l.funders));
       await getStaticAsset('alert').then(setAlertInfo);
     };
-    fetchStaticAssets();
+    fetchStaticAssets().then(() => setTimeout(() => setLoading(false), 1000));
```

Comment:
Why do you need to wait another second to toggle the loading state?
```suggestion
    fetchStaticAssets().then(() => setTimeout(() => setLoading(false), 1000));
```

Reply:
that was so it didn't flash awkwardly. it really only shows for 1000ms (+ time to send over the loading page <100ms)